go/oasis-node/cmd/signer/ledger/ledger.go

```diff
@@ -0,0 +1,38 @@
+// Package ledger implements the ledger signer sub-commands.
+package ledger
+
+import (
+	ledger "github.com/zondax/ledger-oasis-go"
+
+	cmdCommon "github.com/oasislabs/oasis-core/go/oasis-node/cmd/common"
+	ledgerSigner "github.com/oasislabs/oasis-core/go/oasis-node/crypto/signature/signer/ledger"
+)
+
+var (
+	ledgerCmd = &cobra.Command{
+		Use:   "ledger",
+		Short: "interact with Ledger devices",
+	}
+
+	listCmd = &cobra.Command{
+		Use:   "list_devices",
+		Short: "list available devices by address",
+		Run:   doLedgerList,
+	}
+)
+
+func doLedgerList(cmd *cobra.Command, args []string) {
+	if err := cmdCommon.Init(); err != nil {
+		cmdCommon.EarlyLogAndExit(err)
+	}
+
+	ledger.ListOasisDevices(ledgerSigner.SignerDerivationPath)
```

Comment:
Should this wrapper (that uses a specific derivation path) live in the `go/common/crypto/signature/signer/ledger` instead of here? Just call it `ListDevices` and make it take no arguments.

Reply:
Ah good idea!